CVE-2021-41773/cvex.yml

```diff
@@ -0,0 +1,6 @@
+blueprint: ubuntu2204-ubuntu2204
+ubuntu1:
+  playbook: ubuntu1.yml
+ubuntu2:
+  playbook: ubuntu2.yml
+  command: "python3 /opt/exploit.py -u http://192.168.56.3:80"
```

Comment:
Please use %ubuntu1% instead of 192.168.56.3, do not hardcode IP addresses

Reply:
Updated